lib/ansible/module_utils/network/dellos9/dellos9.py

```diff
@@ -2,7 +2,7 @@
 # (c) 2015 Peter Sprygada, <psprygada@ansible.com>
 # (c) 2017 Red Hat, Inc
 #
-# Copyright (c) 2016 Dell Inc.
+# Copyright (c) 2017 Dell Inc.
```

Comment:
I don't believe the date here should be updated like this.  From my understanding, this date indicates how far back the copyright is valid for.  Bumping this forward, instead of leaving as is, or supplying a range, means you are no longer claiming copyright during the 2016 year.